.github/workflows/build-docker.yaml

```diff
@@ -32,6 +30,6 @@ jobs:
       id: push
       uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83
       with:
-        push: ${{ github.event_name == 'push' }}
+        push: ${{ startsWith(github.ref, 'refs/tags/') }} # only push to ghcr.io on tags
         tags: ghcr.io/${{github.repository}}:latest
```

Comment:
If we're pushing tags then we should probably also tag the docker image with the corresponding github tag